backend/docs/developers/setup-local-env.rst

```diff
@@ -43,19 +44,36 @@ You need the following libraries and/or programs:
        npm run build
 
 #. Run the migrations with ``src/manage.py migrate``
+
+.. note::
+
+       If you get an error about the ``django.contrib.gis`` module,
+       you need to install the GDAL library and/or enable the extension or give superuser privileges. See the requirements in the Dockerfile.
 #. Create a superuser to access the management interface: ``src/manage.py createsuperuser``
 #. To set environment variables settings, vreate a ``.env`` file. You can use and modify the provided example:
 
   .. code:: bash
 
     cp dotenv.example .env
 
+#. Generate the translation files:
+ .. code:: bash
+
+    src/manage.py makemessages -a
+    src/manage.py compilemessages --locale nl
+
 #. Run the development server with ``src/manage.py runserver``
 
+Optionally You can load initial zaak data to populate your database. Do note, that this is a timely process (~30-60 minutes):
+
+ .. code:: bash
+
+    src/manage.py cache_zaken
```

Comment:
Before this can work, you need to configure your services. Here https://github.com/maykinmedia/open-archiefbeheer/blob/main/backend/docs/developers/setup-local-env.rst#open-zaak it describes how to start openzaak locally and there is a fixture to configure the openzaak services. 
But maybe we should also add to it a selectielijst service fixture :thinking:

Reply:
(Also with the local openzaak it takes ~one minute to sync)